github/PullRequest.py

```
@@ -128,10 +128,10 @@ class ReviewComment(TypedDict):
     path: str
     position: NotRequired[int]
     body: str
-    line: NotRequired[int]
-    side: NotRequired[str]
     start_line: NotRequired[int]
+    line: NotRequired[int]
```

This should fail with 422 if start_side is not set for a multi-line comment.